ledger/src/shred.rs

```diff
@@ -131,13 +129,12 @@ pub const MAX_CODE_SHREDS_PER_SLOT: usize = MAX_DATA_SHREDS_PER_SLOT;
 // 2. Merkles are chained
 // 3. No retransmit signature (only included for last batch)
 pub const fn get_data_shred_bytes_per_batch_typical() -> u64 {
-    let capacity =
-        match merkle::ShredData::const_capacity(PROOF_ENTRIES_FOR_32_32_BATCH, true, false) {
-            Ok(v) => v,
-            Err(_proof_size) => {
-                panic!("this is unreachable");
-            }
-        };
+    let capacity = match merkle::ShredData::const_capacity(PROOF_ENTRIES_FOR_32_32_BATCH, false) {
+        Ok(v) => v,
+        Err(_proof_size) => {
```

Comment:
You can .unwrap() here. Since const_capacity is const, compiler should be able to prove it never panics. Also we are calling const function with const args here, so it will literally never panic.